backend/pkg/database/database.go

```diff
@@ -318,6 +350,10 @@ GROUP BY vulnerabilities.id,
          pv.fix_version;
   `
 
+	createVulnerabilitiesViewIndex = `
+CREATE UNIQUE INDEX vulnerabilities_view_id ON vulnerabilities_view (id, package_id);
```

Comment:
why here the index is on `(id, package_id)` and not only `id` as all other?

Reply:
Refreshing conrurrently a view requires a unique index, but if we use only the id for the created index it will seem duplicated in the case of vulnerabilities.
```
ERROR: duplicate key value violates unique constraint "vulnerabilities_view_id" (SQLSTATE 23505)
```